### PR TITLE
Event Landing Page for both Web and Mobile

### PR DESCRIPTION
**Pull request by maryam-olabisi**
Event landing page for both web and mobile

### CODE REVIEW

File: public/styles/events.css [hunk 1]
```diff
@@ -0,0 +1,225 @@
+.event_header {
+    opacity: 1;
+}
+
+.event_title{
+    text-align: left;
+    font: Bold 50px Montserrat;
```
Comment (mazma1) [inline public/styles/events.css:7]: I see you used the "Montserrat" font family a lot. There's this [issue](https://github.com/she-code-africa/SCA-Website-V2/issues/14) for making it the website's main font. Can you swap this out for us at once so you don't get to repeat it a lot?

Comment (mazma1) [reply public/styles/events.css:7]: Also, there is this [variables](https://github.com/she-code-africa/SCA-Website-V2/blob/develop/public/styles/variables.css) file for common styles. You'll see some color codes you used (change it to use the variables instead. You can also add other color codes so we can limit repetition as much as possible

Comment (maryam-olabisi) [reply public/styles/events.css]: Sub texts are now bigger, additional white spaces have been added between sub sections, variable names for the color codes used, and the app's default font face is now Montserrat as in Issue #14

Comment (maryam-olabisi) [reply public/styles/events.css]: Screenshot from 2020-08-05 10-34-22

**Comment by temilolakutelu** [comment]
Kindly resolve merge conflicts

**Pull request merged**